test/unit/metrics/test_dashboard_dictionary.py

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+from fairlearn.metrics import create_dashboard_dictionary
+from fairlearn.metrics._group_metric_set import create_group_metric_set
+
+
+def test_create_dashboard_dictionary_smoke():
+    y_true = [0, 1, 0, 1, 0, 1, 0, 1]
+    y_pred = [0, 0, 1, 1, 0, 0, 1, 1]
+    groups = [1, 1, 1, 1, 2, 2, 2, 2]
+    m_name = "my model"
+    g_name = "my group"
+    model_type = 'binary_classification'
+
+    actual = create_dashboard_dictionary(
+        model_type,
+        y_true,
+        {m_name: y_pred},
+        {g_name: groups})
+
+    expected = create_group_metric_set(model_type,
+                                       y_true,
+                                       [y_pred],
+                                       [groups],
+                                       model_titles=[m_name],
+                                       sensitive_feature_names=[g_name])
+
+    assert expected == actual
+    assert actual['schemaType'] == 'dashboardDictionary'
+    assert actual['schemaVersion'] == 0
+
+
+def test_create_dashboard_dictionary_multiple_models_multiple_sensitive_features():
+    y_true = [0, 1, 0, 1, 1, 1, 1, 1, 0, 1, 0]
+
+    y_p1 = [0, 1, 1, 1, 0, 0, 0, 0, 1, 1, 1]
+    y_p2 = [1, 1, 1, 0, 0, 0, 1, 1, 1, 1, 1]
+    y_p3 = [0, 0, 0, 0, 0, 0, 1, 1, 1, 1, 0]
+    m_1 = "M1"
```

Comment:
What if my keys are of a type other than string? Does that still work? [in python, probably, but I know you're looking at a e2e flow]

Reply:
Which keys do you mean? The `y_pred` or the model name?

Reply:
model name
I essentially tried to confirm whether you considered that keys in a dictionary could be something other than string, and whether that works with UI code when transformed into JSON etc.

Reply:
I don't do any checking. I could check that it's a string... or force it to one. But if users wanted type checking, would they be using Python?

Reply:
Sorry, I'm not expressing myself well here. I just wanted to ask whether there's any way a user could do something unintentionally wrong and pass a non-string type that would break the visualizations because it expects a string. I'm probably being paranoid :-)

Reply:
That would ultimately depend on how JS deserialised the JSON file. Since I believe the JS approach to type checking is even more relaxed than Python's, I wouldn't want to hazard a guess at what would happen.